code/src/components/CounterAndSorter.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import Select from 'react-select';
+
+// Styling
+import { InnerFlexWrapper, InfoText } from '../styling/GlobalStyling';
+
+// ----------------------------------------------------------------
+
+export const CounterAndSorter = ({ showingCategory, setShowingCategory }) => {
+  const allTodos = useSelector((store) => store.todos.tasks);
+
+  //Filtering by completed/not completed
+  const nonCompletedTasks = allTodos.filter((todo) => !todo.isCompleted);
+  const completedTasks = allTodos.filter((todo) => todo.isCompleted);
+
+  // Amount of prioritized tasks
+  const amountOfPrioritized = nonCompletedTasks.filter(
+    (task) => task.prio === true
+  ).length;
+
+  // Count how many incomplete tasks there are in selected category
+  const amountOfItems = (category) => {
+    return nonCompletedTasks.filter((task) => task.category === category)
+      .length;
+  };
+
+  // Category options
+  const options = [
+    { value: 'All', label: `All (${nonCompletedTasks.length})` },
```

Comment:
This was related to the package you installed, right? Would be great to learn more about that 👍